core/src/main/java/org/geysermc/geyser/registry/populator/ItemRegistryPopulator.java

```diff
@@ -218,20 +199,17 @@ public static void populate() {
                 }
 
                 String bedrockIdentifier;
-                // 1.19.70+
-                if (palette.getValue().protocolVersion() >= Bedrock_v575.CODEC.getProtocolVersion() && mappingItem.getBedrockIdentifier().equals("minecraft:wool")) {
+                if (mappingItem.getBedrockIdentifier().equals("minecraft:wool")) {
```

Comment:
can't we yeet this + logs workaround entirely and just fix the mappings? since pre 1.19.80 is getting dropped anyways

Reply:
probably, i'll come back to it (or someone else can)